manticore/platforms/linux.py

```diff
@@ -447,20 +450,42 @@ def _init_cpu(self, arch):
         self._function_abi = CpuFactory.get_function_abi(cpu, 'linux', arch)
         self._syscall_abi = CpuFactory.get_syscall_abi(cpu, 'linux', arch)
 
-    def _execve(self, program, argv, envp):
+    def _find_symbol(self, name):
+        symbol_tables = [s for s in self.elf.iter_sections()
+                         if isinstance(s, SymbolTableSection)]
+
+        if not symbol_tables and self.elffile.num_sections() == 0:
+            return None
```

Comment:
I think this condition is redundant. If `symbol_tables` will be `[]` we will get into `return None` anyway.

Reply:
indeed